lib/api/CInferenceModelDefinition.cc

```diff
@@ -90,10 +91,20 @@ auto toJson(const std::string& value, CSerializableToJson::TRapidJsonWriter& wri
     return result;
 }
 
-auto toJson(double value, CSerializableToJson::TRapidJsonWriter&) {
+auto toJson(const std::string& value, CSerializableToJson::TGenericLineWriter& writer) {
+    rapidjson::Value result;
+    result.SetString(value, writer.getRawAllocator());
+    return result;
+}
+
+auto toJson(double value) {
     return rapidjson::Value{value};
 }
 
+auto toJson(double value, CSerializableToJson::TRapidJsonWriter&) {
+    return toJson(value);
+}
```

Comment:
Is this needed now?  (Maybe originally you used the same template `addToJsonArray` for both `double` and `string`.)

Reply:
I removed all those `toJson` functions as we don't need templates anymore.